src/main/java/model/UserModel.java

```diff
@@ -19,6 +19,20 @@ public UserModel() {
 		this.daoFactory = new DaoFactory();
 	}
 	
+	public boolean checkInteger(String number) {
+		String regex = "^[0-9]+$";
+		return number.matches(regex);
+	}
+	
+	public boolean checkBoolean(String bool) {
+		if(bool.equals("true") || bool.equals("false")) {
+			return true;
+		}
+		else {
+			return false;
+		}
+	}
```

Comment:
I do not really see the use of this check method, java parseboolean will work for any type of strings even if it is null in which caseit will return false. The only thing could perhaps be checked with boolean is in case it is 1 or something similar as java will convert only "true" to True any differrent value or number will be false